lib/mi_bridges/driver/job_income_more_about_page.rb

```diff
@@ -3,7 +3,7 @@
 module MiBridges
   class Driver
     class JobIncomeMoreAboutPage < BasePage
-      TITLE = /More About .*'s Job/
+      TITLE = /More About (.*)'s Job/
```

Comment:
oooh should we make this regex change to the other "More" pages as well?

Reply:
We should - I'll make note and open another PR.